isic_archive/tasks/image.py

```diff
@@ -43,6 +43,36 @@ def ingestImage(self, imageId):
         Image().save(image)
         return
 
+    # Store the image stripped of exif metadata
+    try:
+        originalFileStreamResponse = self.session.get(f'file/{imageFile["_id"]}/download')
```

Comment:
`originalFileStreamResponse.seek(0)` and reuse the existing object, rather than downloading it again.